lib/controllers/frontend/spree/user_passwords_controller.rb

```diff
@@ -5,6 +5,9 @@ class Spree::UserPasswordsController < Devise::PasswordsController
   include Spree::Core::ControllerHelpers::Common
   include Spree::Core::ControllerHelpers::Order
   include Spree::Core::ControllerHelpers::Store
+  if Gem.loaded_specs['spree_i18n']
```

Comment:
@MatthewKennedy it would be better to add a check `defined?(SpreeI18n::ControllerLocaleHelper)`